core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel_Common.hpp

```diff
@@ -205,7 +205,7 @@ struct ParallelReduceSpecialize<FunctorType, Kokkos::RangePolicy<PolicyArgs...>,
     // architecture in the future.
```

Comment:
Do you still need this comment?
Also with these changes it looks like you don't need `MAX_ACTIVE_THREADS` any more in which case you should get rid of it.

Reply:
Yeah got rid of `MAX_ACTIVE_THREADS`. Not needed anymore.